### PR TITLE
Add patient_tribal_citizenship to pdi schema

### PR DESCRIPTION
**Pull request by whytheplatypus**
This PR ...

## Changes
- patient_tribal_citizenship field has been added to the SimpleReport schema

### Testing
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [N/A] Added tests?

### Specific Security-related subjects a reviewer should pay specific attention to
None

### Process
- [N/A] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
*List GitHub issues this PR fixes*
- #1746 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema [hunk 1]
```diff
@@ -17,6 +17,9 @@ elements:
   - name: patient_suffix
     csvFields: [{name: Patient_suffix}]
 
+  - name: patient_tribal_citizenship
+    csvFields: [{name: Tribal_affiliation}]
```
Comment (nullflux) [inline prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema:21]: **suggestion:** Can we call this column `Patient_tribal_affiliation`, so it retains the same specificity as the other patient columns?

**Review by nullflux — COMMENTED** [review]
LGTM re: the documentation; I've left a suggestion about the column naming

File: prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema (hunk 1 above)
Comment (jimduff-usds) [inline prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema:21]:
```suggestion
    csvFields: [{name: Patient_tribal_affiliation}]
```

Adam's change.  (don't forget to update the darn .md file too...)

**Review by jimduff-usds — APPROVED** [review]
David, could you tweak the name per Adam's request, regenerate the .md, then merge?

**Pull request merged**